src/components/organisms/Accordion/AccordionHeader/AccordionHeader.tsx

```diff
@@ -34,14 +44,15 @@ const Title = styled.span`
 export interface IAccordionHeader {
   /** determines the position of the chevron icon */
   isOpen: boolean;
+  textSize?: TTextSize;
 }
 
 const AccordionHeader: FunctionComponent<IAccordionHeader> = React.forwardRef<HTMLButtonElement, IAccordionHeader>(
-  ({ children, isOpen, ...rest }, ref) => (
+  ({ children, isOpen, textSize = 'lead', ...rest }, ref) => (
```

Comment:
no need for default here or we can make it optional?

Reply:
I didn't mean to make it a required prop, it should be optional.

Reply:
my bad, it's all good. I looked at the text props by mistake